services/Makefile

```diff
@@ -11,33 +11,32 @@ check: depends lint migrate
 	fi;
 
 lint: depends
-	$(NODE) npm run eslint
+	npm run eslint
 
-fix-formatting:
-	$(NODE) npm run eslint -- --fix
+fix-formatting: depends
+	npm run eslint -- --fix
 
-unit:
-	$(COMPOSE) run -e NODE_ENV=test -e ERROR_LOGGING_FILE=/srv/evergreen/error-logging.json --rm node npm run jest
+unit: depends
+	NODE_ENV=test npm run jest
 
 debug-unit:
-	$(COMPOSE) run -e NODE_ENV=test -e ERROR_LOGGING_FILE=/srv/evergreen/error-logging.json --rm -p 9229:9229 node \
-		node --inspect-brk=0.0.0.0:9229 node_modules/.bin/jest --runInBand --bail --forceExit test/
+	NODE_ENV=test node --inspect-brk=9229 jest -i -b --forceExit test/
 
 acceptance:
-	$(COMPOSE) run -e NODE_ENV=test -e ERROR_LOGGING_FILE=/srv/evergreen/error-logging.json --rm node npm run acceptance
+	NODE_ENV=test npm run acceptance
 
 debug-acceptance:
-	$(COMPOSE) run -e NODE_ENV=test -e ERROR_LOGGING_FILE=/srv/evergreen/error-logging.json --rm -p 9229:9229 node \
-		node --inspect-brk=0.0.0.0:9229 node_modules/.bin/jest --runInBand --bail --forceExit acceptance/
+	NODE_ENV=test node --inspect-brk=9229 jest -i -b --forceExit acceptance/
 
 watch:
-	$(COMPOSE) run -e NODE_ENV=test -e ERROR_LOGGING_FILE=/srv/evergreen/error-logging.json --rm node jest --bail --watchAll
+	NODE_ENV=test jest --bail --watchAll test/
```

Comment:
Surprising, did you intend/need to add this `test/` remainder?

Reply:
Of course :smiling_imp: